yelp_package/dockerfiles/itest/chronos/Dockerfile

```diff
@@ -14,20 +14,20 @@
 
 FROM ubuntu:trusty
 
-RUN apt-get update && apt-get -y install apt-transport-https
+RUN DEBIAN_FRONTEND=noninteractive apt-get update > /dev/null && DEBIAN_FRONTEND=noninteractive apt-get -y install apt-transport-https > /dev/null
```

Comment:
That was going to be my approach, but then I found this discussion: https://github.com/moby/moby/issues/4032#issuecomment-34597177

Seems like they discourage setting it with ENV, because then things will be weird when doing a `docker run -it <> bash`